README.md

```diff
@@ -182,6 +182,10 @@ can be used in your `~/.laptop.local`.
 See the [wiki](https://github.com/thoughtbot/laptop/wiki)
 for more customization examples.
 
+Some things to note.
+------------
+- It's advicable to not install [`NVM`](https://github.com/nvm-sh/nvm) on your machine and opt to using [`ASDF`](https://github.com/asdf-vm/asdf) as this is what the package uses as it's default package manager.
```

Comment:
```suggestion
- It's advisable to not install [`NVM`](https://github.com/nvm-sh/nvm) on your machine and opt to use [`ASDF`](https://github.com/asdf-vm/asdf) as this is what the packager uses as it's default package manager.
```

Reply:
Do you want to move this section to the "Requirements" heading at the top of the README? It will be helpful to read this section in advance before starting the App setups.

Reply:
Sure @rakeshpetit , i will make a new PR with this changes.